delivery_readmes/DELIVERY.README.MethylSeq.txt

```diff
@@ -0,0 +1,92 @@
+README
+=======
+
+MethylSeq Best Practice analysis: Delivered files
+-----------------------------------------------------------------
+
+==================================================================
+ProjectID
+==================================================================
+The data delivery comes in a folder named by the NGI project ID. This contains one “00-Reports“ directory, a directory for each sample with its data, as well as the BP analysis data in a folder called “/results.” The root folder also contains the following files:
+
+--ACKNOWLEDGEMENTS.txt
+A file describing what to include in any publications using this data.
+
+--miscellaneous.lst
+A list of the files of the delivery, excluding raw data.
+
+--miscellaneous.md5
+A file to check the integrity of the miscellaneous.lst file.
+
+--SampleN.lst
+A list of the raw data files of SampleN.
+
+--SampleN.md5
+A file to check the integrity of the SampleN.lst file.
+File integrity checks
+---------------------
+It is extremely important to check the integrity of the delivered data using the .md5 files. To do this, run the following:
+md5sum -c file_with_checksums.md5
+For more information, please see https://ngisweden.scilifelab.se/resources/data-delivery/
+
+
+==================================================================
+ProjectID -> 00-Reports
+==================================================================
+The 00-Report folder contains reports with sequencing information about the runs, about the lanes, libraries, and samples. It also contains a MultiQC report.
+
+
+|--ProjectID/
+   |--00-Reports/
+    --ProjectName_lanes_info.txt
+    --ProjectName_library_info.txt
+    --ProjectName_multiqc_report.html
+    --ProjectName_project_summary.html
+    --ProjectName_project_summary.md
+    --ProjectName_sample_info.txt
+    |--manifestFiles/
+           --<NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```

Comment:
This file representation looks a little odd.. Could we do the normal tree output instead? eg:

```suggestion
── ProjectID
   ├── 00-Reports/
   ├── ProjectName_lanes_info.txt
   ├── ProjectName_library_info.txt
   ├── ProjectName_multiqc_report.html
   ├── ProjectName_project_summary.html
   ├── ProjectName_project_summary.md
   ├── ProjectName_sample_info.txt
   └── manifestFiles/
       └── <NGI_ID>.02-FASTQ.<flowcell>.<NGI_ID>_<flowcell-mode>_<lane-number>_manifest.txt
```

Also for the other file representations below..

Reply:
Done!